doc/_includes/channel_interpolation.rst

```diff
@@ -13,7 +13,7 @@ Spherical spline interpolation (EEG)
    can omit the title from the include:
    channel-interpolation-begin-content
 
-In short, data repair using spherical spline interpolation :footcite:`PerrinEtAl1989` consists of the following steps:
+In short, data repair using spherical spline interpolation\ :footcite:`PerrinEtAl1989` consists of the following steps:
```

Comment:
is this a mistake? it looks OK to me in current dev docs

Reply:
The backslash is the recommended approach by sphinxcontrib-bibtex to remove the space before `[idx]`, to match the style of a publication. I can revert if you prefer to leave the extra space, but I personnaly prefer to have `some text[idx]` rather than `some text [idx]`.

Reply:
My vote is to revert. I could be convinced to add them back (everywhere) in a separate PR